packages/verified-fetch/src/utils/parse-url-string.ts

```diff
@@ -89,30 +97,43 @@ export async function parseUrlString ({ urlString, ipns, logger }: ParseUrlStrin
   let cid: CID | undefined
   let resolvedPath: string | undefined
   const errors: Error[] = []
+  let resolveResult: IPNSResolveResult | DNSLinkResolveResult | undefined
+  let ttl: number = 5 * 60 // 5 minutes based on https://github.com/ipfs/boxo/issues/329#issuecomment-1995236409
 
   if (protocol === 'ipfs') {
     try {
       cid = CID.parse(cidOrPeerIdOrDnsLink)
+      ttl = 29030400 // 1 year for ipfs content
```

Comment:
I would avoid introducing the concept of TTL for ipfs content and just leave it undefined. Higher levels should set the cache-control header values based on `protocol`. 
```suggestion
```